opm/output/eclipse/EclipseWriter.cpp

```diff
@@ -548,10 +530,10 @@ void EclipseWriter::writeInit( const NNC& nnc ) {
         fortio.writeKeyword("PERMZ", data);
     }
 
-    if( !nnc.hasNNC() ) return;
+    if( !this->impl->nnc.hasNNC() ) return;
 
     std::vector<double> tran;
-    for( NNCdata nd : nnc.nncdata() ) {
+    for( NNCdata nd : this->impl->nnc.nncdata() ) {
```

Comment:
I know that this has nothing to do with this PR, but it came to my attention here:
Wouldn't it be reasonable to not rely on the compiler to remove the unnecessary copy of NNCData (Two size_t and one double) and use a reference?
```
for( NNCdata& nd : this->impl->nnc.nncdata() ) {
```
It would also not hurt to initialize the container with the correct size beforehand to prevent reallocations:
```
std::vector<double> tran(this->impl->nnc.nncdata().size());
```
Then one could also could use some old school STL algorithms that tend to be forgotten:
```
std::transform(this->impl->nnc.nncdata().begin(), this->impl->nnc.nncdata().size().end(), tran.begin(),
                       [](const NNCData& i){return nnc.trans;}); 
```